### PR TITLE
Benkuhn/readme update

### PR DESCRIPTION
**Pull request by BenJKuhn**
Updating the project readme to provide a bit more context about what the xlang project and related projects are about, and to include missing links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,53 @@
...
+
+### C++
+
+* [C++/WinRT](https://github.com/microsoft/cppwinrt) - A tool that provides C++ support for calling Windows Runtime APIs that has a modern C++ feel.
+
+* [C++ winmd parser](https://github.com/microsoft/winmd) - A C++ parser library for efficiently parsing WinRT metadata.
+
+* [WinRT ABI (C with classes) Header Generation Tool ](https://github.com/microsoft/xlang/tree/master/src/tool/abi) - Tool that generates the C++ representation of the raw "COM-style" interfaces that make WinRT work.
+
+* [C++ /Win32](https://github.com/microsoft/cppwin32) - Tooling that leverages the Win32 metadata to provide a modern C++ development experience for Windows APIs.
```
Comment (jonwis) [inline README.md:23]: Nit - spacing on `C++ /Win32` ?

File: README.md [hunk 2]
```diff
@@ -1,20 +1,53 @@
...
+
+* [Windows Implementation Libraries](https://github.com/microsoft/wil) - A set of helper libraries that provide RAII and other helpful abstractions for many existing SDK headers & types.
+
+### C#
+
+* [C#/WinRT](https://github.com/microsoft/cswinrt) - Interop support for WinRT APIs and .NET 5.
+* [C#/Wi3n2](https://github.com/microsoft/cswin32) - A metadata-based projection of Win32 and COM APIs.
```
Comment (jonwis) [inline README.md:30]: Misspelled `Win32`

File: README.md [hunk 3]
```diff
@@ -1,20 +1,53 @@
...
+
+* [WinRT ABI (C with classes) Header Generation Tool ](https://github.com/microsoft/xlang/tree/master/src/tool/abi) - Tool that generates the C++ representation of the raw "COM-style" interfaces that make WinRT work.
+
+* [C++ /Win32](https://github.com/microsoft/cppwin32) - Tooling that leverages the Win32 metadata to provide a modern C++ development experience for Windows APIs.
+
+* [Windows Implementation Libraries](https://github.com/microsoft/wil) - A set of helper libraries that provide RAII and other helpful abstractions for many existing SDK headers & types.
+
+### C#
+
+* [C#/WinRT](https://github.com/microsoft/cswinrt) - Interop support for WinRT APIs and .NET 5.
```
Comment (jonwis) [inline README.md:29]: Markdown style - sometimes you put newlines between bullet-list items, here you don't.

**Review by jonwis — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,20 +1,54 @@
...
+
+* [C#/Win32](https://github.com/microsoft/cswin32) - A metadata-based projection of Win32 and COM APIs.
+
+### Rust
+
+* [Rust/WinRT](https://github.com/microsoft/winrt-rs) - Tooling that creates WinRT, Win32 and COM APIs for the Rust programming language.
```
Comment (kennykerr) [inline README.md:35]:
```suggestion
* [Rust for Windows](https://github.com/microsoft/windows-rs) - Tooling that creates WinRT, Win32 and COM APIs for the Rust programming language.
```

**Review by kennykerr — APPROVED** [review]

**Pull request merged**